### DISCUSSION TITLE
How do I get translation, non-English with Python Library?

### DISCUSSION THREAD

**Discussion by geekinchief22** [openai/whisper#1309] [Q&A]
I have a 15 second clip of Portuguese audio (langtest.wav) that transcribes and translates perfectly when I use the CLI

whisper langtest.wav --task translate

However, I am writing a Python script to perform this task and it's not translating or even transcribing properly. Here's what I have:

```
import whisper
model = whisper.load_model("tiny.en") # I have tried this with the base model as well with no luck
myfile = "langtest.wav"
asr_result = model.transcribe(myfile, language='Portuguese', task='translate')
```
The end result is that I get a word salad that's in English. This is the case, even if I take out both the language and task parameters. It doesn't seem to detect that this non-English at all.

**Comment by glangford** [comment]
You are using the wrong model
> model = whisper.load_model("tiny.en") # I have tried this with the base model as well with no luck

This model is for English only. Try a model without ".en".

See also
* https://github.com/openai/whisper/discussions/1185

**Comment by geekinchief22** [comment]
Thanks. That worked!
